src/sentry/api/event_search.py

```diff
@@ -522,7 +522,7 @@ def __init__(self, config=None, params=None, builder=None):
             self.builder = UnresolvedQuery(
                 dataset=Dataset.Discover,
                 params=self.params,
-                config=QueryBuilderConfig(functions_acl=FUNCTIONS.keys()),
+                config=QueryBuilderConfig(functions_acl=list(FUNCTIONS.keys())),
```

Comment:
`list(d.keys())` is usually better written as `list(d)` -- but perhaps the type should be updated to `Container[str]` ?

Reply:
I'll go with `list(d)` for now